Microsoft.Toolkit.Parsers/Markdown/MarkdownDocument.cs

```diff
@@ -374,15 +390,28 @@ public LinkReferenceBlock LookUpReference(string id)
                 return null;
             }
 
-            LinkReferenceBlock result;
-            if (_references.TryGetValue(id, out result))
+            if (_references.TryGetValue(id, out LinkReferenceBlock result))
             {
                 return result;
             }
 
             return null;
         }
 
+        /// <summary>
+        /// Adds Custom Scheme to the list of existing schemes
+        /// </summary>
+        public void AddSchemes(string schemelist)
```

Comment:
This is not necessary here and can be a private method in the MarkdownTextBlock.

Reply:
It's not clear that the schemelist needs to be comma delimited and developers can just add items directly to the list without having to use this function.